### PR TITLE
ROMS on ARM

### PR DESCRIPTION
**Pull request by Yikun**

### CODE REVIEW

File: ROMS.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+### Installing GNU 9.3
+
+
+    yum install -y centos-release-scl
+    yum install -y devtoolset-9-gcc
+    yum install -y devtoolset-9-gcc-c++
+    yum install -y devtoolset-9-binutils
+    scl enable devtoolset-9 bash
+    echo "souce /opt/rh/devtoolset-9/enable" >> /etc/profile
+### Installing Open MPI
```
Comment (martin-g) [inline ROMS.md:40]: Add Open MPI to the table with dependencies

File: ROMS.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+## Configuring the Compilation Environment
+### Installing GNU 9.3
+
+
+    yum install -y centos-release-scl
+    yum install -y devtoolset-9-gcc
+    yum install -y devtoolset-9-gcc-c++
+    yum install -y devtoolset-9-binutils
+    scl enable devtoolset-9 bash
+    echo "souce /opt/rh/devtoolset-9/enable" >> /etc/profile
```
Comment (martin-g) [inline ROMS.md]: s/souce/source/

**Review by martin-g — COMMENTED** [review]
Same improvements could be done as in the other PRs

File: ROMS.md (hunk 1 above)
Comment (qinluyao761) [reply ROMS.md:40]: done

File: ROMS.md (hunk 2 above)
Comment (qinluyao761) [reply ROMS.md]: done

**Pull request merged**